openpype/plugins/publish/collect_host_name.py

```diff
@@ -14,7 +14,7 @@ class CollectHostName(pyblish.api.ContextPlugin):
     """Collect avalon host name to context."""
 
     label = "Collect Host Name"
-    order = pyblish.api.CollectorOrder - 1
+    order = pyblish.api.CollectorOrder - 0.5
```

Comment:
This is out of the collector range as well. `-0.5 < range < 0.5`

Reply:
Nope `-0.5` is included by `inrange` function in pyblish.